dbt_meshify/dbt_projects.py

```diff
@@ -43,6 +44,32 @@ def __init__(
             source.relation_name: unique_id for unique_id, source in self.sources().items()
         }
 
+        self._graph = None
+
+        self._changes: Dict[str, str] = {}
+
+
+    @staticmethod
+    def _load_graph(manifest: Manifest) -> Graph:
+        """Generate a dbt Graph using a project manifest and the internal dbt Compiler and Linker."""
+
+        from dbt.compilation import Compiler
```

Comment:
is having imports within a method a stylistic choice, or are there performance benefits (or neither)?

Reply:
I'm doing it more from a performance perspective. Top-of-file imports result in all of the project's dependencies being loaded from disk at the time of interpretation. Normally this is great, but for a CLI-based tool, it can create some lag during invocation. In our case, the `dbt-core` package is a little on the big side, so I'm trying to get into the habit of only importing locally when possible.